tools/state-manager/test_restore_runner.py

```
@@ -0,0 +1,121 @@
+        # Ensure cleanup runs even if the import below raises.
+        self.addCleanup(self._module_patcher.stop)
+
+        # Drop any cached restore_runner so the import below sees the stubs.
+        self._prior_restore_runner = sys.modules.pop("restore_runner", None)
+        self.addCleanup(self._restore_prior_module)
+
+        from restore_runner import RestoreRunner  # noqa: E402  pylint: disable=wrong-import-position
+
+        self.RestoreRunner = RestoreRunner
+
+    def _restore_prior_module(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_restore_prior_module`

Add return type annotation: `None`

(ANN202)